src/main/resources/liquibase/workbench_changelog/17_4_1.xml

```diff
@@ -28,5 +28,25 @@
 		</sql>
 	</changeSet>
 
-</databaseChangeLog>
+	<changeSet author="lkovacic" id="v17.4.1-3">
+		<preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="4">
+                SELECT count(*)
+				FROM role_permission rp
+						 INNER JOIN role r ON rp.role_id = r.id
+						 INNER JOIN permission p ON rp.permission_id = p.permission_id
+				WHERE r.name = 'Readonly' AND p.name IN ('IMPORT_GERMPLASM', 'SEARCH_GERMPLASM', 'IMPORT_GERMPLASM_UPDATES', 'GERMPLASM_LABEL_PRINTING');
```

Comment:
I think we should use role id = 4 instead of the name (or use both), because the role name is modifiable. 
What do you think @nahuel-soldevilla

Reply:
Done!